src/actions/filter-criteria-actions.js

```diff
@@ -55,24 +55,24 @@ export const saveFilterCriteria = (filterCriteria) => async (dispatch, getState)
 
 export const deleteFilterCriteria = (filterCriteriaId) => async (dispatch, getState) => {
 
-    const accessToken = await getAccessTokenSafely();    
+    const accessToken = await getAccessTokenSafely();
 
     const params = {
         access_token: accessToken
     };
 
-    dispatch (startLoading());
+    dispatch(startLoading());
 
     return deleteRequest(
         null,
-        createAction(FILTER_CRITERIA_DELETED)({filterCriteriaId}),
+        createAction(FILTER_CRITERIA_DELETED)({ filterCriteriaId }),
         `${window.PERSIST_FILTER_CRITERIA_API}/api/v1/filter-criterias/${filterCriteriaId}`,
         null,
         authErrorHandler
     )(params)(dispatch).then(() => {
-            dispatch(stopLoading());
-        }
-    );    
+        dispatch(stopLoading());
```

Comment:
what happen if there is an error ?
stop loading should be on finally i do think